.vscode/settings.json

```diff
@@ -20,4 +20,7 @@
     "[rust]": {
         "editor.defaultFormatter": "rust-lang.rust-analyzer",
     },
+    "rust-analyzer.linkedProjects": [
+        "./tun/Cargo.toml"
```

Comment:
Is this needed? `rust-analyzer` should ideally pick up the `tun` crate from the Rust workspace

Reply:
It also looks like it disables auto-discovery:

> Disable project auto-discovery in favor of explicitly specified set of projects.

Reply:
Indeed! This is not needed, and it turns out to be a local development environment issue resulting in VSCode prompting me to add this.